pyblish_deadline/plugins/select_deadline_write_nodes.py

```diff
@@ -61,4 +62,11 @@ def process(self, context):
                 plugin_data = plugin_data.copy()
                 plugin_data['WriteNode'] = node.name()
 
+                try:
+                    components = {node['fcompname'].getValue(): {}}
+                    instance.set_data('ftrackComponents', value=components)
```

Comment:
This seems quite workflow specific, and it makes our render instances fail on conforming to ftrack. Any way of doing this outside, with a different plugin?